File: Python/Day2/Task2.py
def  get_Grade(first_Score, second_Score, third_Score):

	average = first_Score + second_Score + third_Score;
	average = average / 3;

	if (average >= 90 and average <= 100):
		print("Grade is: A")

	elif(average >= 80 and average < 90):
			print("Grade is: B")

	elif (average >= 70 and average < 80):
			print("Grade is: C")

	elif (average >= 60 and average < 70):
			print("Grade is: D")

	else:
		print("F");

	return average;


firstNumber = 56
secondNumber = 45
thirdNumber = 70

print(get_Grade(firstNumber, secondNumber, thirdNumber))

File: Java/Day2/Task2.java
import java.util.Scanner;
public class Scores{
public static void main(String... args){

Scanner scanner = new Scanner(System.in);

System.out.println("Enter first score: ");
int firstNumber = scanner.nextInt();

System.out.println("Enter second score: ");
int secondNumber = scanner.nextInt();

System.out.println("Enter third score: ");
int thirdNumber = scanner.nextInt();

System.out.println(getGrade(firstNumber, secondNumber, thirdNumber));
	}





public static int getGrade(int firstScore, int secondScore, int thirdScore){
	
	int average = firstScore + secondScore + thirdScore;
average = average / 3;

if (average >= 90 && average <= 100){
	System.out.println("Grade is: A");
	}
else if (average >= 80 && average < 90){
		System.out.println("Grade is: B");
}
else if (average >= 70 && average < 80){
	System.out.println("Grade is: C");
}
else if (average >= 60 && average < 70){
	System.out.print("Grade is: D");
}	
  else{
	System.out.println("F");
	}
	

return average;
}






}